extensions/wikia/TriviaQuizzes/TriviaQuizzes.setup.php

```diff
@@ -0,0 +1,24 @@
+<?php
+$wgExtensionCredits['other'][] = [
+	'name' => 'Trivia Quizzes',
```

Comment:
Something is funky with the indentation in this file. Can you make sure you're using Tab indentation for all the PHP files?

Reply:
Ran through stylize.php 0e890bbd5de6b30e08aaa907fe0da845389ad320